From: Sam Ravnborg <sam@ravnborg.org>
Date: Sat, 8 Sep 2018 11:11:04 +0200
Subject: [PATCH] dt-binding: panel: add Seiko 70WVW2T 7" panel

Add bindings for Seiko Instruments 7" TFT LCD
Include new sii vendor prefix.
sii matches the web-page name: https://www.sii.co.jp and
was already used by an existing panel binding

Signed-off-by: Sam Ravnborg <sam@ravnborg.org>
Cc: Thierry Reding <thierry.reding@gmail.com>
Cc: Rob Herring <robh+dt@kernel.org>
Cc: Mark Rutland <mark.rutland@arm.com>
---
 .../bindings/display/panel/sii,70wvw2t.txt          | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/display/panel/sii,70wvw2t.txt

diff --git a/Documentation/devicetree/bindings/display/panel/sii,70wvw2t.txt b/Documentation/devicetree/bindings/display/panel/sii,70wvw2t.txt
new file mode 100644
index 000000000000..e9da0a24fe66
--- /dev/null
+++ b/Documentation/devicetree/bindings/display/panel/sii,70wvw2t.txt
@@ -0,0 +1,21 @@
+Seiko Instruments Inc. 7.0" WVGA (800 x RGB x 480) TFT with Touch-Panel
+
+Required properties:
+- compatible: should be "sii,70wvw2t".
+- vcc-supply: phandle to 3v3 digital regulator.
+
+Optional properties:
+- backlight: phandle for the backlight control.
+
+Example:
+
+	panel {
+		compatible = "sii,70wvw2t";
+		backlight = <&backlight>;
+		vcc-supply = <&reg_lcd_3v3>;
+		port {
+			panel_in: endpoint {
+				remote-endpoint = <&display_out>;
+			};
+		};
+	};
